packages/frontend/pages/strategies.tsx

```diff
@@ -258,7 +258,7 @@ const Strategies: React.FC = () => {
               </div>
               {supportedNetwork && (
                 <div className={classes.tradeCard}>
-                  {!currentEthValue.isZero() && !isQueued ? <MigrationNotice /> : null}
```

Comment:
just added this back. Can you remove this change?

Reply:
okay will do